Lib/imaplib.py

```diff
@@ -1390,19 +1399,17 @@ def print_log(self):
 
 
 class Idler:
-    """Iterable context manager: start IDLE & produce untagged responses
+    """Iterable context manager: start IDLE & produce untagged responses.
 
     An object of this type is returned by the IMAP4.idle() method.
     It sends the IDLE command when activated by the 'with' statement, produces
     IMAP untagged responses via the iterator protocol, and sends DONE upon
     context exit.
 
-    Iteration produces (type, datum) tuples.  They slightly differ
-    from the tuples returned by IMAP4.response():  The second item in the
-    tuple is a single datum, rather than a list of them, because only one
-    untagged response is produced at a time.
+    Iteration produces (type, [data, ...]) tuples, as described in the IMAP4
+    class documentation.
 
-    Note: The name and structure of this class are internal interfaces,
+    Note:  The name and structure of this class are internal interfaces,
```

Comment:
Stray space?

Reply:
I'm sure I have more right to claim being old-fashioned. :-)

I prefer single-space -- double-spacing between sentences is controversial nowadays, and the Python docs team has long ago decided not to do this any more. (But we're not "fixing" it in old code either!)

I agree that the docstrings could be a bit smaller -- compare nearby existing docstrings. It seems the classes have long docstrings but the methods typically have shorter ones. This is very much a matter of personal taste though (some projects have strict conventions for docstrings, but Python is pretty uneven across the stdlib).

Reply:
Thanks. Even in the days when I was religiously typing two spaces after sentence-ending punctuation, I would not do that after a colon.

See you at next year's PyCon in Pittsburgh!